webapp/javascript/components/TimelineChart/clamp.ts

```diff
@@ -0,0 +1,12 @@
+function clamp(min: number, value: number, max: number) {
```

Comment:
nit but having the actual `value` in the middle is weird, either put it in the front or at the end

putting at the end makes it easier to do partial application

```js
const myClamp = clamp.bind(null, floor, celling);
myClamp(myValue)
```
then when the pipeline operator is finally implemented (https://tc39.es/proposal-pipeline-operator/)
```f#
 value 
  |> myClamp
```

but let's not get too ahead of ourselves haha